src/azure-cli/azure/cli/command_modules/appconfig/_validators.py

```diff
@@ -193,6 +193,13 @@ def validate_key(namespace):
         raise CLIError("Key cannot be empty.")
 
 
+def validate_resolve_keyvault(namespace):
+    if namespace.resolve_keyvault:
+        identifier = getattr(namespace, 'destination', None)
+        if identifier and identifier != "file" and namespace.resolve_keyvault:
```

Comment:
>and namespace.resolve_keyvault [](start = 47, length = 30)

nit:  namespace.resolve_keyvault check is not required again as you are already checking that on line 197

Reply:
Sure, will fix. 

---
In reply to: [405713427](https://github.com/Azure/azure-cli/pull/12893#discussion_r405713427) [](ancestors = 405713427)